app/controllers/sign_up/registrations_controller.rb

```diff
@@ -1,5 +1,5 @@
 module SignUp
-  class RegistrationsController < Devise::RegistrationsController
+  class RegistrationsController < ApplicationController
```

Comment:
I don't think we can do this unless we copy over the stuff that we still need from the Devise Registrations Controller.